messages/logs/cells/messages.go

```diff
@@ -0,0 +1,13 @@
+package cells
+
+var (
+	Usage            = "cells"
+	ShortDescription = "Displays cells console logs"
+	LongDescription  = "Displays cells console logs"
+	FlagHelp         = "Displays more information about the logs cells command"
+	FlagPretty       = "Displays logs in a prettified way"
+	FlagFunctionId   = "ID of the function you wish the see the logs for; if not informed, logs for all functions will be displayed"
```

Comment:
```suggestion
	FlagFunctionId   = "ID of the function for which you wish to see the logs; if not provided, logs for all functions will be displayed."
```

Reply:
@gabriel-azion will create another PR after this, with the final messages... these ones are just placeholders :)

Reply:
ok.